common-services-api/pom.xml

```diff
@@ -12,10 +12,15 @@
     <packaging>bundle</packaging>
 
     <properties>
-      <vendor-driver.version>1.0.0.0</vendor-driver.version>
-      <pentaho-code.version>11.0.0.0</pentaho-code.version>
+      <org.osgi.core.version>4.3.1</org.osgi.core.version>
     </properties>
     <dependencies>
+        <dependency>
+            <groupId>org.osgi</groupId>
+            <artifactId>org.osgi.core</artifactId>
+            <version>${org.osgi.core.version}</version>
```

Comment:
Do not explicitly use the version here. Use the managed dependency from the parent pom.

Reply:
Replaced with parent pom's version